Libraries/MiscDrivers/Touchscreen/tsc2007.h

```diff
@@ -0,0 +1,173 @@
+/******************************************************************************
+ *
+ * Copyright (C) 2023 Analog Devices, Inc. All Rights Reserved.
+ *
+ * This software is proprietary and confidential to Analog Devices, Inc. and
+ * its licensors.
+ *
+ ******************************************************************************
+ *
+ * Copyright 2023 Analog Devices, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************************/
```

Comment:
Update copyright in this newly added file.

Reply:
Updated.